modules/ipi-install-configuring-host-network-interfaces-in-the-install-config.yaml-file.adoc

```diff
@@ -2,26 +2,52 @@
 //
 // installing_bare_metal_ipi/ipi-install-installation-workflow.adoc
 
-[id="configuring-host-network-interfaces-in-the-install-config-yaml-file_{context}"]
-= (Optional) Configuring host network interfaces in the `install-config.yaml` file
+:_content-type: PROCEDURE
+[id="configuring-host-network-interfaces-in-the-install-config.yaml-file_{context}"]
+= Optional: Configuring host network interfaces in the `install-config.yaml` file
 
-During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState. To use the `networkConfig` configuration setting, you must provide an NMState YAML configuration.
+During installation, you can set the `networkConfig` configuration setting in the `install-config.yaml` file to configure host network interfaces using NMState.
 
 [IMPORTANT]
 ====
-Before configuring host network interfaces, review the OpenShift Container Platform 4.10 release notes.
+Host network interface configuration is set during the installation and provisioning process and includes advanced options, such as setting static IPs per host. Before configuring host network interfaces, review the {product-version} 4.10 release notes. 
 ====
 
-See link:https://nmstate.io/examples.html#interfaces-ethernet[NMState] for additional examples of the NMState syntax.
+.Prerequisites
 
+* You configured a `PTR` DNS record with a valid hostname for each node with a static IP address.
+
+* You installed the `jq` package.
+
+.Procedure
+
+. Optional: Consider saving the `networkConfig` YAML syntax to a file.
+
+.. Test the `networkConfig` YAML syntax by using the NMState command-line interface. Test the `networkConfig` YAML syntax before including it in the `install-config.yaml` file, because the installation program will not check the NMState YAML syntax. 
```

Comment:
This seems redundant.

Reply:
@awolffredhat thanks for catching that! Here's a possible rewrite. Let me know if this makes more sense:

> .. Test the `networkConfig` YAML syntax before including it in the `install-config.yaml` file. The installation program will not check the NMState YAML syntax.
> 
> .. Run the following command on the NMState command-line interface:

Reply:
Yes. This does make more sense. thanks.